packages/site/cms-site/src/iframebridge/PreviewOverlayElement.tsx

```diff
@@ -14,11 +15,14 @@ export const PreviewOverlayElement = ({ element }: Props) => {
     const isHovered = element.adminRoute === iFrameBridge.hoveredAdminRoute;
 
     return (
-        <Root
+        <div
             key={element.adminRoute}
-            $showBlockOutlines={iFrameBridge.showOutlines}
-            $blockIsSelected={isSelected}
-            $isHoveredInBlockList={isHovered}
+            className={clsx(
+                styles.root,
+                iFrameBridge.showOutlines && !isHovered && styles.showBlockOutlines,
```

Comment:
Why do you check for hover here? We didn't do this before

Reply:
This is the same logic that previously was done in the styled component:

https://github.com/vivid-planet/comet/blob/main/packages/site/cms-site/src/iframebridge/PreviewOverlayElement.tsx#L77